src/popup/router/pages/Send.vue

```diff
@@ -242,7 +242,7 @@ export default {
           let tx = {
             popup:false,
             tx: {
-              amount:this.form.amount,
+              amount:amount,
```

Comment:
This change leads to wrong display of entered amount in Send Page
![image](https://user-images.githubusercontent.com/8629465/72157053-a815ea00-33bf-11ea-81f4-deb79482d460.png)
![image](https://user-images.githubusercontent.com/8629465/72157080-b5cb6f80-33bf-11ea-9339-a164f71a245e.png)
Amount in second picture should be **1 AE**

Reply:
Fixed in https://github.com/vmangelovv/aepp-waellet/commit/972a6c713f8da8a665eac47952e8c6f660ca7768